CMakeLists.txt

```diff
@@ -90,13 +90,6 @@ message(STATUS "   PYTHON_INCLUDE_DIRS = ${PYTHON_INCLUDE_DIRS}")
 message(STATUS "   PYTHON_LIBRARIES    = ${PYTHON_LIBRARIES}")
 message(STATUS "   Python_EXECUTABLE   = ${Python_EXECUTABLE}")
 
-# The building of the schemas requires some extra python packages which may or
-# may not be installed. If they aren't we provide an option to add them from
-# a python egg.
-if (BUILD_USDMAYA_SCHEMAS)
-    include(cmake/jinja.cmake)
-endif()
-
```

Comment:
@seando-adsk  This was a bit of a surprise since this code never is called when the clean build happens. 

BUILD_USDMAYA_SCHEMAS is an option that is set in AL.  This value is not cached until AL's CmakeLists is called which happens later.

Reply:
Agree, I had to take it out eventually since I was getting bunch of weird issues with Python 3 when doing incremental builds. Let's revisit this together.

Reply:
I realized that this needs to remain here. From the Maya build and our internal ecg-maya-usd repo we are setting this flag "BUILD_USDMAYA_SCHEMAS=ON" and providing the jinja/markupsafe locations. The jinja.cmake will check to see if jinja2 and markupsafe are part of the python executable. If not, then it will add them to the python path from the locations provided.

Reply:
Hey @seando-adsk see 389bc58645baebaf3fc54211d76430428ad4f713